packages/loopback/test/acceptance/routing/routing.acceptance.ts

```diff
@@ -90,6 +94,68 @@ describe('Routing', () => {
     expect(result).to.have.property('body', 'TestApp');
   });
 
+  it('creates a new child context for each request', async () => {
+    const app = givenAnApplication();
+    app.bind('flag').to('original');
+
+    const spec = givenOpenApiSpec()
+      .withOperationReturningString('put', '/flag', 'setFlag')
+      .withOperationReturningString('get', '/flag', 'getFlag')
+      .build();
+
+    @api(spec)
+    class FlagController {
+      constructor(@inject('ctx') private ctx: Context) {
```

Comment:
For the purpose of testing new instances, can you cache `ctx` to the class and assert the new one is not the same as the cached one?